app/models/location.rb

```diff
@@ -5,6 +5,11 @@ class Location < ApplicationRecord
   belongs_to :subdivision, class_name: "Location", optional: true
   belongs_to :city, class_name: "Location", optional: true
 
+  validates :lat, inclusion: -90..90, if: :mass_validation_enabled?
```

Comment:
cc @jshoe

Reply:
Hm, I think during the design we were allowing for lat and lng to be null for edge cases

Reply:
Ok that's interesting. Currently all values in prod are not null. Could you describe the edge cases so I can add them as comments to the columns?

Reply:
I would write: Lat and lng may be null if the location provider is missing coordinates, as opposed to placing a location at (0,0)

Reply:
Ok sounds good. And the downstream locations code will handle nulls by filtering them out?

Reply:
Correct